app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/components/PlanWizardInstancePropertiesStepTable.js

```diff
@@ -105,7 +105,20 @@ class PlanWizardInstancePropertiesStepTable extends React.Component {
       const clusterId = additionalData.rowData.ems_cluster_id;
       const tenantId = destinationTenantIdsBySourceClusterId[clusterId];
       const tenant = tenantId && tenantsWithAttributesById[tenantId];
-      const options = tenant ? tenant[optionsAttribute] : [];
+      const allFitForVM = () => {
+        const { bestFitFlavors } = this.props;
```

Comment:
Little nit, can we do this destructuring up in [line 103](https://github.com/ManageIQ/manageiq-v2v/blob/14392c6ee943abf39c4d8c5f87501ab5fd47a43f/app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/components/PlanWizardInstancePropertiesStepTable.js#L103)